scip.proto

```diff
@@ -97,8 +97,8 @@ message Document {
 //   <manager>              ::= same as above, use the placeholder '.' to indicate an empty value
 //   <package-name>         ::= same as above
 //   <version>              ::= same as above
-//   <descriptor>           ::= <package-descriptor> | <type> | <term> | <method> | <type-parameter> | <parameter> | <meta>
-//   <package-descriptor>   ::= <name> '/'
+//   <descriptor>           ::= <module> | <type> | <term> | <method> | <type-parameter> | <parameter> | <meta>
```

Comment:
What do you think about “namespace” instead of module?

Reply:
I think that's OK too. I've made that change, including a place which I missed earlier with `IdentifierModule`.